kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/TopicsResource.java

```diff
@@ -132,6 +134,37 @@ public void getTopic(
     AsyncResponses.asyncResume(asyncResponse, response);
   }
 
+  @PATCH
+  @Path("/{topicName}")
+  @Consumes(MediaType.APPLICATION_JSON)
+  @Produces(MediaType.APPLICATION_JSON)
+  @PerformanceMetric("v3.topics.partitions")
+  @ResourceName("api.v3.topics.partitions")
+  public void updatePartitionCount(
```

Comment:
nit: It would be good to have a UT here, can be done as a follow-up.

Reply:
Ooo, I had some of them, they missed the git add.  Added now :)